src/daft-connect/src/translation/literal.rs

```diff
@@ -0,0 +1,61 @@
+use eyre::{bail, WrapErr};
+use spark_connect::expression::{literal::LiteralType, Literal};
+
+// todo(test): add tests for this esp in Python
+pub fn to_daft_literal(literal: Literal) -> eyre::Result<daft_dsl::ExprRef> {
+    let Some(literal) = literal.literal_type else {
+        bail!("Literal is required");
+    };
+
+    match literal {
+        LiteralType::Array(_) => bail!("Array literals not yet supported"),
+        LiteralType::Binary(bytes) => Ok(daft_dsl::lit(bytes.as_slice())),
+        LiteralType::Boolean(b) => Ok(daft_dsl::lit(b)),
+        LiteralType::Byte(b) => {
+            // todo(correctness): is this signed or unsigned?
+            let b = i8::try_from(b).wrap_err_with(|| format!("Byte value {b} is out of range"))?;
+            let b = i32::from(b);
+
+            Ok(daft_dsl::lit::<i32>(b))
+        }
```

Comment:
i'd just `bail` here for now. I don't think this is super high prio

Reply:
are we sure? we are basically doign the same thing with shorts

Reply:
I'd prefer adding the variants to `LiteralValue` as we have kernels for those dtypes, we just don't have it in `LiteralValue` yet. So imo, it's fine just skipping the ones that we don't have a 1-1 mapping with, and then making an issue to add `LiteralValue::<unsupported_variant>`

Reply:
ok resolving but note we still are converting f32 -> f64... is this ok? f32 is used a lot

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/izEhZtMLX1KwSAhfusD5/526a6926-39a1-471d-a040-c7eabc901c00.png)